docs/how-to-guides/advanced-user-guides/enable-experimental-features/rancher-on-arm64.md

```diff
@@ -37,7 +37,7 @@ version's release notes in the following two ways:
 
 - Create custom cluster and adding ARM64 based node(s)
   - Kubernetes cluster version must be 1.12 or higher
-  - CNI Network Provider must be [Flannel](../../../faq/container-network-interface-providers.md#flannel)
+  - CNI Network Provider must be Flannel
```

Comment:
The link is still valid per another comment I made about Flannel's availability in RKE2.